src/databroker_elasticsearch/converters.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+"""\
+Functions for converting values that are exported to Elasticsearch.
+"""
+
+# handle registration of converter functions ---------------------------------
+
+def register_converter(f, name=None):
+    "Decorator to mark up some function as a converter."
+    nm = f.__name__ if name is None else name
+    _converters[nm] = f
+    return f
+
+_converters = {}
+
+
+def getconverter(name):
```

Comment:
why do we have a function which duplicates ``__getitem__``?

Reply:
To allow more meaningful error message for invalid name argument (not yet there).
It also separates converter-registry interface from the actual storage implementation.
We are free to rename `_converters` or replace it with some other storage code in the future as these are just internals of the `converters` module.